style.css

```diff
@@ -128,3 +131,263 @@ h1 {
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #091e42;
+}
+
+.button {
+  display: flex;
+  align-items: center;
+}
+
+.button li {
+  text-align: center;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  background-color: #fff;
+  width: 70px;
+  margin-right: 30px;
+}
+
+.button-dark li {
+  text-align: center;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #fff;
+  width: 70px;
+  margin-right: 30px;
+}
+
+.button-orange a {
+  color: #fff;
+}
+
+.button li a {
+  color: #091e42;
+  height: 20px;
+  border: 1px solid #8993a4;
+  padding: 10px 12px;
+}
+
+.button-dark li a {
+  padding: 10px 12px;
+  background: rgba(255, 255, 255, 0.24);
+  color: #fff;
+}
+
+.button li a:hover {
+  border: 1px solid #8993a4;
+  background: #fff5e1;
+  color: #f55800;
+}
+
+.button-dark {
+  display: flex;
+  align-items: center;
+  color: #091e42;
+}
+
+.button-dark li a:hover {
+  border: 1px solid #8993a4;
+  background: #fff5e1;
+  color: #f55800;
+}
+
+.button-orange {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #fff;
+  background-color: #ff6b00;
+  width: 95%;
+  height: 40px;
+  margin-top: 0;
+  padding-top: 20px;
+}
+
+.small-mg {
+  padding: 12px;
+  gap: 20px;
+  width: 124px;
+  height: 20px;
+}
+
+h4 {
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+}
+
+p {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.about {
+  background: #1c1a19;
+}
+
+.about-list {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.about h2 {
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 72px;
+  line-height: 88px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+}
+
+.flex-paragraph {
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+}
+
+.button-resume {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #fff;
+  background-color: #ff6b00;
+  width: 100%;
+  padding: 5px;
+}
+
+#content-abt {
+  background-image: url("assets/illustration-about-me-mobile.png");
+  background-position: bottom right;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 563px;
+  padding: 3em;
+}
+
+#hr {
+  width: 85%;
+}
+
+.about-title {
+  font-family: "Inter", sans-serif;
+  font-style: bold;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+}
+
+.about-li {
```

Comment:
- [ ] The ```languages```, ```skills```, and ```framework``` items should be centered, please implement this and stick to the design as much as possible

|Your design|figma design|
|-|-|
|![Screenshot_20220527_192818](https://user-images.githubusercontent.com/25227022/170771206-ae8af759-bb25-430c-a741-6789c6785983.png)|![Screenshot_20220527_192921](https://user-images.githubusercontent.com/25227022/170771224-c25e9982-675d-40d8-9e5d-0cdfeff46658.png)|